configuration_controller/tests/requirements.txt

```diff
@@ -1,7 +1,7 @@
-coverage~=5.5
```

Comment:
Maybe try to maintain alphabetical order of python packages in requirements file?

Reply:
done